{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -65,6 +65,10 @@ include = ["CHANGELOG.md", "README.md"]
 
 [tool.poetry.dependencies]
 python = "^{{ cookiecutter.python_major_version }}.{{ cookiecutter.python_minor_version }}"
+{%- if cookiecutter.__project_type=="agent_plugin" %}
+monkey-agentpluginapi = "{{ cookiecutter._monkeyagentpluginapi_version }}"
+monkey-types = "{{ cookiecutter._monkeytypes_version }}"
+{%- endif %}
```

Comment:
These should be dev dependencies. They will not be packaged with the plugin, but are provided by the Agent.